interface/main/tabs/main.php

```diff
@@ -361,19 +369,44 @@ function isEncounterLocked( encounterId ) {
 $('#updater-options').click(function () {
     $('#updater-settings-iframe').iziModal('open');
 });
-
+var timer;
+var secondtimer;
 $('#updater-icon').hover(function () {
     $(this).css("border", "2px solid #000");
     $('#developer-mode').fadeIn(100);
     $('#updater-options').fadeIn(100);
+}, function(){
+    timer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
+        $('#updater-options').fadeOut(100);
+        $('#updater-icon').css("border", "none");
+    }, 3000);
+});
+$('#updater-icon-wrapper').hover(function(){}, function(){
+    secondTimer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
```

Comment:
excellent piece of jquery.... works pretty fine

Reply:
I never caught onto this happening before.  I think I should have seen something in the inspector.

Reply:
Wondering if we are setting developer mode by default for superusers ("administrators").